snap/info.go

```diff
@@ -228,6 +228,11 @@ func UserSnapDir(home string, name string, opts *dirs.SnapDirOptions) string {
 	return filepath.Join(home, snapDataDir(opts), name)
 }
 
+// UserExposedHomeDir returns the snap's directory in the exposed home dir.
+func UserExposedHomeDir(home string, name string) string {
```

Comment:
nitpick: I'd rename the `name` variable to `snapName`, or given that all other methods use the same convention, keep it as it is bu then change the docstring to explain what `name` can be (the functions above all say "The name can be either a snap name or snap instance name." — but I'm not sure whether this applies here too).